### ISSUE TITLE
Linux: alm starts only as root

### ISSUE THREAD

**Issue by aravantv** [alm-tools/alm#148]
On Linux, starting alm without `sudo` brings "permission denied" errors.

```
bash ~> alm
fs.js:584
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: EACCES: permission denied, open '/var/local/alm/settingsV1.json'
    at Error (native)
    at Object.fs.openSync (fs.js:584:18)
    at Object.fs.writeFileSync (fs.js:1224:33)
    at Object.writeFile (/usr/lib/node_modules/alm/src/server/utils/fsu.js:55:8)
    at setSettings (/usr/lib/node_modules/alm/src/server/disk/settings.js:46:9)
    at Object.addWorkingDir (/usr/lib/node_modules/alm/src/server/disk/settings.js:56:5)
    at Object.setProjectRoot (/usr/lib/node_modules/alm/src/server/disk/workingDir.js:16:14)
    at Object.<anonymous> (/usr/lib/node_modules/alm/src/server/commandLine.js:52:20)
    at Object.getOptions (/usr/lib/node_modules/alm/src/common/utils.js:173:21)
    at Object.<anonymous> (/usr/lib/node_modules/alm/src/server.js:40:20)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
    at require (internal/module.js:20:19)

```

**Comment by basarat** [comment]
Thanks. Sorry for that. Fixed with `0.1.30` :rose:

**Comment by aravantv** [comment]
Works now thanks!
